models/src/main/java/com/vimeo/networking2/params/ModifyVideosInAlbumSpecs.kt

```diff
@@ -0,0 +1,23 @@
+package com.vimeo.networking2.params
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/**
+ * An object that is used to patch video addition and deletion updates to an Album.
+ */
+@JsonClass(generateAdapter = true)
+data class ModifyVideosInAlbumSpecs(
+    /**
+     * The set of videos that should be removed.
+     */
+    @Json(name = "remove")
+    val removeVideoSet: Set<NamedWrapperForRemove>? = null,
```

Comment:
I could make them inner classes, but we can't make them private, because Moshi requires the properties be visible to the adapters (they aren't visible if they are private), and we can't make the properties public but the class type private as that results in a kotlin error. Let me know if you have other ideas about how you want to design this class in Kotlin + Moshi. Personally, it doesn't bother me that these classes are public, but I understand the original design you implemented was user friendly. Perhaps a custom constructor would be suitable to improve the UX. Let me know what you think.